Backend/src/main/java/idealab/api/operations/DropboxOperations.java

```diff
@@ -127,15 +128,14 @@ public PrintJobResponse updateDropboxPath(DropBoxFilePathRequest request) {
 
     LocalDateTime currentTime = LocalDateTime.now();
     String newPath = "/" + currentTime.toLocalTime().toNanoOfDay() + "-" + request.getNewPath();
+    newPath = newPath.endsWith(FILE_EXTENSION) ? newPath : newPath + FILE_EXTENSION;
```

Comment:
@jberns should we be tacking on the file extension? I feel like we should throw an exception if the file isnt in the proper format.

Reply:
Hm good point.  I was thinking for any new uploads we should definitely enforce STL only, but for rename it wouldn't matter.  We could just have this enforced on the frontend and then always throw an error if .stl is missing if that is what you were thinking.

Reply:
I'd rather throw the exception on the backend. I'd rather not run into some unintended results (thinking corrupted files, malicious files, etc).

Reply:
I think lets merge this in because it updates the API to the move and fixes the current problem of accidentally not including .stl, and then we can open a new issue to add an STL rejection errors to:
1. New Print Job Endpoint
2. Update Model Endpoint
3. Rename Model Endpoint

Let me know what you think.

Reply:
I’m okay with that. You wanna create the issues?